diff --git a/catalog/sql/sql.go b/catalog/sql/sql.go
index 8f2f29f1..25fb7258 100644
--- a/catalog/sql/sql.go
+++ b/catalog/sql/sql.go
@@ -754,7 +754,7 @@ func (c *Catalog) ListNamespaces(ctx context.Context, parent table.Identifier) (
 // avoid circular dependency while still avoiding having to export the getUpdatedPropsAndUpdateSummary function
 // so that we can re-use it in the catalog implementations without duplicating the code.
 
-//go:linkname getUpdatedPropsAndUpdateSummary github.com/apache/iceberg-go/catalog.getUpdatedPropsAndUpdateSummary
+//go:linkname getUpdatedPropsAndUpdateSummary github.com/DataDog/iceberg-go/catalog.getUpdatedPropsAndUpdateSummary
 func getUpdatedPropsAndUpdateSummary(currentProps iceberg.Properties, removals []string, updates iceberg.Properties) (iceberg.Properties, catalog.PropertiesUpdateSummary, error)
 
 func (c *Catalog) UpdateNamespaceProperties(ctx context.Context, namespace table.Identifier, removals []string, updates iceberg.Properties) (catalog.PropertiesUpdateSummary, error) {
